src/elexmodel/models/ConformalElectionModel.py

```diff
@@ -185,8 +186,8 @@ def get_unit_prediction_interval_bounds(
         # are the same accross train_data, conformalization_data and nonreporting_units
         nonreporting_units_features = interval_featurizer.generate_holdout_data(x_all[self.n_train :])  # noqa: E203
 
-        nonreporting_lower_bounds = lower_qr.predict(nonreporting_units_features)
-        nonreporting_upper_bounds = upper_qr.predict(nonreporting_units_features)
+        nonreporting_lower_bounds = lower_qr.predict(nonreporting_units_features.values).flatten()
+        nonreporting_upper_bounds = upper_qr.predict(nonreporting_units_features.values).flatten()
```

Comment:
Interesting, I didn't realize there was a change in the predicted output data structure that would necessitate `.flatten()`s here (and elsewhere) 🤔

Reply:
yes! This is because of the changes we made in elex solver for the quantile regression. We now have this for-loop where quantile regression iterates over the different quantiles and adds the coefficients to a list. That new outer list is what adds the extra dimension that we want to get rid of here.